spec/features/update_case_information_from_allocation_page_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature "Update case information from allocation page", type: :feature do
```

Comment:
Now I'm looking at it – it feels like this test might be covering the same thing as the other feature spec?

This feature spec is nicer because it doesn't depend on VCR. Maybe we could remove the test from the Case Information feature spec if it's already covered here.

I can only see one context block in this file, but I think the ticket MO-735 wanted us to support this behaviour for two user journeys:

1. **For an existing allocation** Allocation information => change Case Info => allocation information
2. **For a new allocation** Allocate a POM => change Case Info => Allocate a POM

I think there might actually be a third journey, too:
3. **Reallocating a POM on an existing allocation**: Allocation information => Reallocate POM => change Case Info => Reallocate POM

So I was expecting to see more context blocks in this spec.

Reply:
Thanks @ollietreend, yes you are right. I think I really struggled understanding those flow diagrams but now you have written it out that makes lots of sense. Thanks you.